js_modules/dagit/packages/ui/src/components/TokenizingField.tsx

```diff
@@ -69,17 +69,14 @@ export function tokenizedValueFromString(
   str: string,
   providers: SuggestionProvider[],
 ): TokenizingFieldValue {
-  const parts = str.split(':');
-  const token = parts[0];
+  const [token = '', value = ''] = str.split(':');
+
   if (findProviderByToken(token, providers)) {
-    if (parts.length === 2) {
-      return {token, value: parts[1]};
-    }
-    // Jobs can be `pipeline:mode` tuples, so join the "value" tokens back together.
-    if (token === 'job') {
-      return {token, value: parts.slice(1).join(':')};
+    if (token && value) {
+      return {token, value};
     }
   }
```

Comment:
Pretty sure this is dead now, since `pipeline` and `job` are treated separately. Also I'm glad to remove Dagit product-specific code from core UI.

Reply:
Yep I think this makes sense 👍